openshift/wps-api-base/openshift/build.yaml

```diff
@@ -1,21 +1,21 @@
-apiVersion: v1
+apiVersion: template.openshift.io/v1
 kind: Template
 metadata:
   creationTimestamp: null
-  name: uvicorn-gunicorn-fastapi
+  name: wps-api-base
 labels:
   app: ${NAME}${SUFFIX}
   phase: build
-  app.kubernetes.io/name: uvicorn-gunicorn-fastapi
+  app.kubernetes.io/name: wps-api-base
   app.kubernetes.io/managed-by: template
-  app.kubernetes.io/version: "python3.8"
+  app.kubernetes.io/version: "ubuntu.22.04"
 parameters:
   - name: NAME
     value: wps-api-base
   - name: SUFFIX
   - name: VERSION
     description: Output version
-    value: "python3.9-latest"
+    value: "ubuntu.22.04-latest"
```

Comment:
According to Openshift 201, using an image tagged with `-latest` is not recommended practice because of the uncertainty around versioning - e.g., the version of the image could change over time as new images get tagged `-latest`. I don't know how much I care, just sharing some wisdom.

Reply:
I mean - then it would just become "ubuntu.22.04" - which probably has the same amount of meaning...